pages/home.js

```diff
@@ -92,79 +107,87 @@ export default function Home(props) {
 
   return (
     <>
-      <h1>{props.content.header}</h1>
-      <p>{props.content.description}</p>
+      <h1 className="mb-4">{props.content.header}</h1>
+      {!response ? <p>{props.content.description}</p> : undefined}
```

Comment:
We could pop this p tag down into the same condition on line 120

Reply:
fair points on both suggestions, changed!